src/components/smallComponents/ContributorProfile.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { Image, Placeholder, Transformation } from "cloudinary-react"
+
+function ContributorProfile(props) {
```

Comment:
Please use arrow function and directly destructure contributors in the argument `function ContributorProfile({contributors}) {` and then use it like this `const { name, profilePhoto, title } = contributors`
Nothing wrong with your code, just to maintain the coding style